src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/__tests__/step_index_pattern.test.js

```diff
@@ -23,166 +20,75 @@ const savedObjectsClient = {
 };
 const goToNextStep = () => {};
 
-describe('StepIndexPattern', () => {
-  it('should render normally', () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    expect(component).toMatchSnapshot();
-  });
-
-  it('should render the loading state', () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    component.setState({ query: 'k', isLoadingIndices: true });
-
-    expect(component).toMatchSnapshot();
-  });
-
-  it('should render some indices', async () => {
-    const component = shallow(
-      <StepIndexPattern
-        allIndices={allIndices}
-        isIncludingSystemIndices={false}
-        esService={esService}
-        savedObjectsClient={savedObjectsClient}
-        goToNextStep={goToNextStep}
-      />
-    );
-
-    const instance = component.instance();
+const createComponent = props => {
+  return shallow(
+    <StepIndexPattern
+      allIndices={allIndices}
+      isIncludingSystemIndices={false}
+      esService={esService}
+      savedObjectsClient={savedObjectsClient}
+      goToNextStep={goToNextStep}
+      {...props}
+    />
+  );
+};
 
-    await instance.onQueryChanged({
-      target: { value: 'k' }
+describe('StepIndexPattern', () => {
+  describe('step 1', () => {
```

Comment:
Does this extra describe feel necessary? When I see this, I assume there is a step 2 or something

Reply:
You're right, this is totally superfluous.